app/controllers/ideas_controller.rb

```diff
@@ -19,7 +19,9 @@ def index
 
   # GET /ideas/1
   # GET /ideas/1.json
-  def show; end
+  def show
+    @comments = Comment.where(idea_id: @idea.id)
```

Comment:
You can do `@idea = Idea.find(params[:id]).includes(:comments)` to load the idea along with its comments in one database query. Then in the view you can do `@idea.comments.each do |comment|`